CONTRIBUTING.md

```diff
@@ -0,0 +1,79 @@
+# Contributing to Boxwise
+
+First off: **thank you**. You will be helping build software that distributes clothes and food to thousands of refugees and other people in need. It gives comfort and dignity to people in a vulnerable situation. That can't be said for most open source contributions.
+
```

Comment:
"That can't be said for most open source contributions" Is this a bit too blunt or arrogant sounding? Maybe it would be better to phrase it something like:

"Not only will you be contributing to the open source community / ecosystem of open source projects, but your contributions will directly benefits thousands of people who are stuck in a vulnerable system." 

These are semantics though, and probably a more philosophical than practical point.

Reply:
changed.